projects/components/src/breadcrumbs/breadcrumbs.component.test.ts

```diff
@@ -53,12 +57,21 @@ describe('BreadcrumbsComponent', () => {
     expect(spectator.queryAll('.breadcrumbs .breadcrumb').length).toBe(3);
   });
 
-  it('should show icon only on the first crumb', () => {
+  it('should show icon on the first crumb', () => {
```

Comment:
This is pre-existing tech debt in this file so it's better to be consistent (or fix the whole file), but just as an FYI - we generally use the jest syntax `test` instead of the jasmine syntax `it`

Reply:
Ok. It's clear.